src/app/ui/dynamics_popup.h

```diff
@@ -37,8 +37,11 @@ namespace app {
 
     tools::DynamicsOptions getDynamics() const;
     void setOptionsGridVisibility(bool state);
-    void loadDynamicPref(ToolPreferences* toolPref);
+    void loadDynamicsPref(Preferences* pref);
```

Comment:
```suggestion
    void loadDynamicsPref();
```
And move `#include "app/pref/preferences.h"` to the `dynamics_popup.cpp`